### PR TITLE
postmatch

### PR DESCRIPTION
**Pull request by borzoi388**

### CODE REVIEW

**Comment by azaleacolburn** [comment]
I'm going to wait until #7 is merged before merging this one, because the merge conflict resolution will be easier that way.

File: src/routes/+page.svelte
Comment (0xda157) [inline src/routes/+page.svelte]: generally alerts aren't used, and in this case it's defiantly redundant b/c the placeholder text already provides that info to the user

File: src/routes/scout/endgame/+page.svelte
Comment (0xda157) [inline src/routes/scout/endgame/+page.svelte]: these two should be camelCase or snake_case for better readability

File: src/routes/scout/endgame/+page.svelte
Comment (0xda157) [inline src/routes/scout/endgame/+page.svelte]: 1. items start is redundant b/c it's the default
2. align-top is used for inline or table-cell box and has no effect on flexboxes, you're looking for items-start (which is the default on flexboxes)
3. it would be better to use min-h-dvh in case everything doesn't fit on the screen

File: src/routes/scout/endgame/+page.svelte
Comment (0xda157) [inline src/routes/scout/endgame/+page.svelte]: margin 0 is the default when using tailwind

Comment (0xda157) [reply src/routes/scout/endgame/+page.svelte]: would also be better to apply gap-2 to parent div instead of using mr-2

**Review by 0xda157 — CHANGES_REQUESTED** [review]
Overall looks good, just a few small things that should be change. Also keep in mind that the colors will need to be changed once #9 is merged to main.

File: src/routes/scout/[team_data]/Postmatch.svelte
Comment (0xda157) [inline src/routes/scout/[team_data]/Postmatch.svelte:17]: should probably set the [accent color](https://tailwindcss.com/docs/accent-color) for this page

File: src/routes/scout/[team_data]/Postmatch.svelte
Comment (0xda157) [inline src/routes/scout/[team_data]/Postmatch.svelte:18]: all of the borders on everything doesn't match the other pages and looks out of place (this can be fixed in a future pr)

File: src/routes/scout/[team_data]/Postmatch.svelte
Comment (0xda157) [inline src/routes/scout/[team_data]/Postmatch.svelte:21]: all of the mb, mt, mx or my should be replaced with flex gap

File: src/routes/scout/[team_data]/Postmatch.svelte
Comment (0xda157) [inline src/routes/scout/[team_data]/Postmatch.svelte:52]: remove width from label and put `justify-evenly` on parent (setting width on a flex child is almost always a bad idea)

File: src/routes/scout/[team_data]/Postmatch.svelte
Comment (0xda157) [inline src/routes/scout/[team_data]/Postmatch.svelte:53]: > all of the mb, mt, mx or my should be replaced with flex gap

**Review by 0xda157 — CHANGES_REQUESTED** [review]
just need to fix some css nitpicks

File: src/routes/scout/[team_data]/Postmatch.svelte
Comment (0xda157) [reply src/routes/scout/[team_data]/Postmatch.svelte:52]: also text-left is the default so it can be removed

**Review by 0xda157 — APPROVED** [review]

**Pull request merged**
